va/compat_win32.h

```diff
@@ -28,6 +28,7 @@
 #include <winsock.h>
 #include <io.h>
 #include <time.h>
+#include <stdlib.h>
```

Comment:
leave in-source comment why we include it here - people most likely will miss the reason when reading the code